packages/next/build/webpack-config.ts

```diff
@@ -1223,6 +1223,7 @@ export default async function getBaseWebpackConfig(
     },
     watchOptions,
     output: {
+      ['compareBeforeEmit' as any]: false,
```

Comment:
`as any` because types didn't contain `output.compareBeforeEmit`

Reply:
@sokra are there any performance drawbacks to doing this? Seems the comparison is failing in this case and it's not emitting the file when switched between compilers.

Reply:
Possibly this won't be needed after this is fixed https://vercel.slack.com/archives/C035J346QQL/p1659704191110839